code/addons/vitest/src/plugin/viewports.ts

```diff
@@ -50,12 +50,12 @@ const parseDimension = (value: string, dimension: 'width' | 'height') => {
 export const setViewport = async (viewportsParam: ViewportsParam = {} as ViewportsParam) => {
   const defaultViewport = viewportsParam.defaultViewport;
 
-  if (!page || !globalThis.__vitest_browser__ || !defaultViewport) {
+  if (!page || !globalThis.__vitest_browser__) {
     return;
   }
 
   const viewports = {
-    ...INITIAL_VIEWPORTS,
+    ...MINIMAL_VIEWPORTS,
```

Comment:
Should this also be included?

https://github.com/storybookjs/storybook/blob/039f2e8c5b85537664267194f5d1d5f299058c56/code/addons/viewport/src/responsiveViewport.tsx#L4